templates/page.html

```diff
@@ -347,12 +347,12 @@ <h3>Developers</h3>
         <div class="release-news">
           <img src="{{ '/static/img/release.png'|asseturl }}" />
           <div class="release-news-body">
-            <h3>REANA 0.9.1</h3>
-            <h6>28 Sep 2023</h6>
-            <p>Visualise PDF and ROOT files in the web interface.</p>
-            <p>Authenticate users with Keycloak.</p>
-            <p>Numerous improvements to cluster performance.</p>
-            <p class="release-news-link"><a href="https://github.com/reanahub/reana/releases/tag/0.9.1">See full release notes</a></p>
+            <h3>REANA 0.9.2</h3>
+            <h6>18 Dec 2023</h6>
+            <p>Remove workflow restart limit.</p>
+            <p>Automatically mount any CVMFS repository.</p>
+            <p>Docker images now available for ARM platforms.</p>
```

Comment:
Deploy REANA clusters on ARM architecture hardware.

_(To keep the same style for all three messages.)